config/environments/staging.rb

```diff
@@ -5,6 +5,19 @@
 
   config.action_mailer.default_url_options = { host: 'codemontage.com' }
```

Comment:
Oops! This host URL is for Production, not Staging :scream:. Please update to earn :trophy:. 
Current behavior is that the emails sent from Staging environment direct the user to a Production password reset page.